students/tri_nguyen/lesson03/mailroom.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  15-Feb-2018
+# ------------------------------------------- #
+
+# Declare variables and data structure
+
+donor_list = [
+    ['William Gates, III', 653784.49, 2, 326892.24],
+    ['Mark Zuckerberg', 16396.10, 3, 5465.37],
+    ['Jeff Bezos', 877.33, 1, 877.33],
+    ['Paul Allen', 708.42, 3, 236.14],
+    ['Tri Nguyen', 100.00, 1, 100.00]
+    ]
+
+
+# Declare functions
+
+
+def display_menu():
+    ''' this function display a menu
+        of 3 actions '''
+    ACTIONS = ('Send a Thank You', 'Create a Report', 'Quit')
+
+    print(''' Select one of the actions below:
+
+    1. {}
+    2. {}
+    3. {}
+    '''.format(*ACTIONS))
+
+
+def send_thanks():
+    ''' this function composes an email thanking
+        the donor for their generous donation '''
+
+    while True:
+
+        full_name = input('Enter a Full Name (or type "list" to get the list of donors): ')
+
+        if full_name.lower() == 'list':
+            for idx, item in enumerate(donor_list):
+                print(idx + 1, '-', donor_list[idx][0])
+            continue
+
+        elif full_name.title() not in [donor_list[idx][0] for idx, item in enumerate(donor_list)]:
```

Comment:
instead have this:
`elif full_name not in [donor[0] for donor in donor_list]`

`enumerate` is not necessary since you only need the items in the list